### ISSUE TITLE
Python 3.11 -  No module named 'UserDict'

### ISSUE THREAD

**Issue by rsirres** [verterok/tritcask#8]
Hi,

When using tritcask 0.3 using python 3.11 you will receive the following error:

```
tritcask.py", line 65, in <module>
    from collections import MutableMapping as DictMixin
ImportError: cannot import name 'MutableMapping' from 'collections'
```

Best,
Raphaël

**Comment by luciotorre** [comment]
can you try this one? https://github.com/verterok/tritcask/pull/9

**Comment by verterok** [comment]
fixed by ca488e41a20a412aa2ec577c12b20abc2be59376
